extensions/git/src/git.ts

```diff
@@ -650,6 +650,11 @@ export class Repository {
 		await this.run(args);
 	}
 
+	async deleteBranch(name: string): Promise<void> {
+		const args = ['branch', '-d', name];
```

Comment:
Maybe we should be explicit with the branch name and use its full form `refs/heads/${name}` instead of simply `name`.